### DISCUSSION TITLE
Very slow plotting on mac OS mojave

### DISCUSSION THREAD

**Discussion by stinyanont** [iraf-community/iraf#159] [Q&A]
I installed pyraf through astroconda and found that while most things work ok, the plotting is unusably slow. For example, when running apall to extract spectra, it would take a good few minutes after I press any key for the graphic display to response. One thing I notice is that the graphic window looks "high quality" compared to what I'm used to on a linux machine at work. I'm not sure if there's some kind of quality or resolution flags that I can adjust to speed things up. 

Have anyone seen this kind of problem before?

**Comment by olebole** [comment]
As far as I know, astroconda uses the original IRAF distribution, which has a lot of problems fixed here. Therefore, the best for you would be to contact the Astroconda people and ask them what to do.
